servicebroker/lib/util/validateUtil.js

```diff
@@ -5,17 +5,17 @@ module.exports = {
       if (keys != null) {
         if (keys instanceof Array) {
           for (var key in keys) {
-            if (typeof (object[keys[key]]) === "undefined" || object[keys[key]] === null) {
-              return false;
+            if (object[keys[key]] == null) {
+              return { valid: false, message: "The value of " + keys[key] + " should not be undefined or null" };
```

Comment:
Rather than go round and round, I have created a chore to clean up the validation.